Tests/Covid19Api.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,49 @@
+using Covid19Api.Models;
```

Comment:
Rename this file from UnitTest.cs to something that makes more sense probably. 'CovidServiceTest'